### ISSUE TITLE
Importing library error

### ISSUE THREAD

**Issue by crowdor** [rmcrackan/Libation#547]
**Describe the bug**
An error message came up when trying to import books for the first time. I just made my account in the settings tab. Login was successful. No books appeared.

**To Reproduce**

![Screenshot 2023-03-25 114200](https://user-images.githubusercontent.com/128896744/227709626-d28d1932-c1fe-48cc-9c98-3b5eebb428dd.png)

**Expected behavior**
Importing books from audible account to library

**Screenshots**
![Screenshot 2023-03-25 114200](https://user-images.githubusercontent.com/128896744/227709626-d28d1932-c1fe-48cc-9c98-3b5eebb428dd.png)

**Platform**
Windows 11
[e.g. Windows 10, Windows 11, Mac, Linux (State distribution)]

**Log Files**
Attach your Libation log file here.
[_Log202303.log](https://github.com/rmcrackan/Libation/files/11068519/_Log202303.log)

**Comment by Mbucari** [comment]
@crowdor this is the same errors as https://github.com/rmcrackan/Libation/issues/543. I'm pretty sure I know what's wrong and that I've fixed it, but I'd like you to help me confirm. Please [download and run this fixed version](https://github.com/Mbucari/Libation/actions/runs/4474423544) and tell me if it fixed your error.

**Comment by rmcrackan** [comment]
This will be included in the next version release :)

**Comment by crowdor** [comment]
I just managed to download and run version 9.4.6.1 for windows classic (I downloaded it from a link on the bottom of the page to which you link in your message). It immediately opened a popup asking me if I wanted to update to version 10.0.2, which has the fix for the library scan bug. However, in libation I now see my library, which was the point, so is it okay if I try it first without further updating?
